### ISSUE TITLE
thirdparty: add pfUI backdrop colors to meters

### ISSUE THREAD

**Issue by moootv** [shagu/pfUI#586]
I tried the newest masterversion in a raid yesterday. Really like it.

I would like to have the same colors in dpsmate and KTM which you define in the pfUI Settings -> Appearance -> Background Color.

Also i would like to have an option to scale KTM to e.g. 120%, to make it bigger (because it is more important in a fight than dpsmate).
You can currently do this in the KTM settings, but then it's conflicting with dpsmate:
https://gyazo.com/a4a28599a9d1cd1273e515458a7a22e2

So basically an option to scale KTM would be nice, while it's scaling dpsmate smaller.

Another idea would be an option to hide dpsmate in combat, while exclusively showing KTM.
If not in combat exclusively show dpsmate.

**Comment by shagu** [comment]
The scale won't happen, because there is no way to resize KTM beside of the scale. One would need to fork KTM and make it automatically respect it's width and height. Right now, If I scale it to 120%, the width of it will be completely broken.

**Comment by shagu** [comment]
This is already done. I forgot to close this issue.
